### PR TITLE
Release v2.6.2-beta2

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

Looking for the PR for the upcoming second beta of the v2.6.2 series? Well you've just found it, congratulations! :tada:

## Release checklist

- [x] ~~Check to see if there are any errors in Sentry that have only occurred since the last production release~~
- [x] Verify that all feature flags are flipped appropriately
 - no feature flags
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
  - no new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.6.2-beta2": [
+      "[Fixed] Remotes referencing forked repos would be removed even when branches from them were available locally - #11266",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Forked repository remotes are no longer removed when there's local branches tracking them - #11266",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+      "[Fixed] Full screen won't start as a white screen anymore - #5631. Thanks @AndreiMaga!",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Avoid bright flash for users of the dark theme when launching the app maximized - #5631. Thanks @AndreiMaga!",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+      "[Fixed] Checking out a pull request from a fork for the first time now correctly identifies the remote branch to create the branch from - #11267",
+      "[Fixed] VSCodium wouldn't be recognized as an editor on Windows - #11252. Thanks @KallePM!"
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] VSCodium is now detected as an editor on Windows - #11252. Thanks @KallePM!"
```

**Review by niik — CHANGES_REQUESTED** [review]
Thanks for putting this together @sergiou87! I had a few small suggestions. See https://github.com/desktop/desktop/blob/development/docs/process/writing-release-notes.md, we try not to describe how the behavior has improved, not what was wrong.

**Comment by sergiou87** [comment]
Noted, thank you!!

**Review by niik — APPROVED** [review]
:sparkles:

**Comment by damecia** [comment]
UNSUSCRIBE ME INMEDIATELY 


Sent from the all new Aol app for iOS


On Wednesday, December 23, 2020, 5:58 AM, Sergio Padrino <notifications@github.com> wrote:




Noted, thank you!!

**Pull request merged**